README.md

```diff
@@ -20,143 +20,16 @@ _Thanks to the [Better Jinja](https://github.com/samuelcolvin/jinjahtml-vscode)
-        <title>{{ page_meta.html_title }}</title>
-        {% if site_settings.favicon_src %}<link rel="shortcut icon" href="{{ site_settings.favicon_src }}" />{% endif %}
-        {{ standard_header_includes }}
-    </head>
-    <body>
-        ${stuff}
-        {{ standard_footer_includes }}
-    </body>
-</html>
-```
-`hublblog`
-```
-{% if is_listing_view %}
-    <!-- Markup for blog listing template -->
-    {% for content in contents %}
-        <h2><a href="{{content.absolute_url}}">{{ content.name }}</a></h2>
-        {% if content.post_list_summary_featured_image %}
-            <a href="{{content.absolute_url}}">
-                <img src="{{ content.post_list_summary_featured_image }}" alt="{{ content.featured_image_alt_text }}">
-            </a>
-        {% endif %}
-        {{ content.post_list_content|safe }}
-    {% endfor %}
-{% else %}
-    <!-- Markup for blog post template -->
-    <h1>{{ content.name }}</h1>
-    <a href="{{ group.absolute_url }}/author/{{ content.blog_post_author.slug }}">{{ content.blog_post_author.display_name }}</a>
-    {{ content.publish_date_localized }}
-    {{ content.post_body }}
-    {% for topic in content.topic_list %}
-        <a href="{{ blog_tag_url(group.id, topic.slug) }}">{{ topic.name }}</a>{% if not loop.last %},{% endif %}
-    {% endfor %}
-    {% blog_comments "blog_comments" overrideable=False, label='Blog Comments' %}
-{% endif %}
-```
-[`request.<variables>`](https://designers.hubspot.com/docs/hubl/hubl-supported-variables#http-request-variables)
-```
-{{ request.cookies }}
-{{ request.domain }}
-{{ request.full_url }}
-etc.
-```
-[`Email Required Template Variables`](https://designers.hubspot.com/docs/hubl/hubl-supported-variables#required-email-template-variables)
-```
-{{ site_settings.company_city }}
-{{ site_settings.company_name }}
-{{ unsubscribe_link }}
-{{ unsubscribe_link_all }}
-etc.
-```
-
-__Custom Module Fields__
-In effort to allow for more streamlined local development of custom modules, snippets have been added that can be added to the fields.json file within a custom module.
-The snippet trigger roughly corresponds with the field names in the design manager.
-For example:  to add a rich text field, you would type
-```
-field.richtext
-```
-To create a group, use
-```
-group.group
-```
-To create a repeater, use
-```
-group.repeater
-```
-all associated fields need to be added into the children array within the snippet output
+## Getting Started
+This extension introduces new file languages: `HTML + HUBL` and `HTML + CSS`. In order to use these new features, you'll want to make sure you're told VSCode to use these langauges. For one-off files, you can simple change the language in the lower right hand corner of your screen. However, we recommend adjusting your file associations under your User or Workspace preferences:
+- In VSCode, press `CMD` + `SHIFT` + `P` to open the command prompt
+- Search for `files.associations`
```

Comment:
This does not work for me. I just get a "No matching commands" when searching for this
![image](https://user-images.githubusercontent.com/6472448/93397984-b2d72780-f848-11ea-974a-0f31a20f5ddd.png)

Maybe we can direct them to open the user/project settings and add it to the JSON? Also a link to https://code.visualstudio.com/docs/getstarted/settings might be useful here.

Reply:
Ah, good catch! LMK what you think about the updated verbiage. I reference the UI because you can easily tab between User/Workspace vs the JSON method which involves two different files in two different locations.